pnpm-lock.yaml

```diff
@@ -3043,7 +3042,6 @@ packages:
     resolution: {integrity: sha512-+1VkjdD0QBLPodGrJUeqarH8VAIvQODIbwh9XpP5Syisf7YoQgsJKPNFoqqLQlu+VQ/tVSshMR6loPMn8U+dPg==}
     engines: {node: '>=14'}
     requiresBuild: true
-    dev: true
```

Comment:
Is it ok that these `dev:true` flags are removed?

Reply:
unsure, but i think its fine.